client/client.go

```diff
@@ -24,13 +24,14 @@ import (
 	"io/ioutil"
 	"log"
 	"net/http"
+	"net/url"
 	"os"
 	"strings"
 	"time"
 )
 
 var (
-	action  = flag.String("action", "deploy", "expand | deploy | list | get | delete | update")
+	action  = flag.String("action", "deploy", "expand | deploy | list | get | delete | update | listtypes | gettype")
```

Comment:
should probably really be listtypeinstances or listinstances with a --type. gettype might do something completely different in the future if there is a real type registry concept.